qa/integration-tests/src/test/java/io/camunda/it/StandaloneSchemaManagerTest.java

```diff
@@ -0,0 +1,323 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.it;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.application.Profile;
+import io.camunda.operate.webapp.api.v1.entities.ProcessInstance;
+import io.camunda.qa.util.cluster.TestStandaloneCamunda;
+import io.camunda.qa.util.cluster.TestStandaloneSchemaManager;
+import io.camunda.search.clients.query.SearchQueryBuilders;
+import io.camunda.search.connect.configuration.ConnectConfiguration;
+import io.camunda.search.connect.es.ElasticsearchConnector;
+import io.camunda.zeebe.broker.system.configuration.ExporterCfg;
+import io.camunda.zeebe.model.bpmn.Bpmn;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration.TestZeebe;
+import io.camunda.zeebe.test.util.asserts.EitherAssert;
+import java.io.IOException;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.Map;
+import java.util.Objects;
+import org.awaitility.Awaitility;
+import org.elasticsearch.client.RestClient;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.elasticsearch.ElasticsearchContainer;
+import org.testcontainers.images.builder.Transferable;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+
+@ZeebeIntegration
+@Testcontainers
+final class StandaloneSchemaManagerTest {
+
+  public static final String ADMIN_USER = "camunda-admin";
```

Comment:
Given our time constraints, only do this if it's really quick, otherwise let's do this as a follow-up: Could you add an additional test-case that verifies that without executing the schema manager beforehand, that running the application with those parameters will fail due to non-existing schema? This would be important to verify that the positive cases you are testing really are running on an application with limited elasticsearch privileges

Reply:
I verified this manually now but would skip adding an actual test for two reasons:
1. It's not super straightforward because the `camunda.start()` doesn't fail with a nice exception
2. We'd add ~20 seconds test runtime again